merlin/models/tf/loader.py

```diff
@@ -334,10 +334,6 @@ def __init__(
         self.sparse_max = sparse_max or {}
         self.sparse_as_dense = sparse_as_dense
 
-    @property
-    def input_schema(self) -> Schema:
```

Comment:
This is unrelated to this PR, but needs to be removed because the datloader now has the input_schema property with a setter in https://github.com/NVIDIA-Merlin/dataloader/pull/86